dao/json/entities/json-applications.js

```diff
@@ -243,6 +243,9 @@ class JsonApplications {
                         self: { href: '/applications/' + appId }
                     }
                 };
+                
+                if(appCreateInfo.description) //since it's optional
+                    newApp.description = appCreateInfo.description.substring(0, 128);
```

Comment:
Shouldn't we have more space here, like at least 1K? I can imagine with support for links and things, 128 characters is much too little.